src/Sets/Zonotope.jl

```diff
@@ -554,29 +554,36 @@ end
 # Zonotope vertex enumeration methods
 # ====================================
 
-function _vertices_list_2D(c::AbstractVector{N}, G::AbstractMatrix{N}; apply_convex_hull::Bool) where {N}
-    if same_sign(G)
-        return _vertices_list_2D_positive(c, G)
-    else
-        # FIXME generalized 2D vertices list function is not implemented yet
-        # See LazySets#2209
-        return _vertices_list_iterative(c, G, apply_convex_hull=apply_convex_hull)
-    end
+@inline function _angles(point::AbstractVector{N})  where{N}
+    (atand(point[2], point[1]) + 360) % 360
 end
 
-function _vertices_list_2D_positive(c::AbstractVector{N}, G::AbstractMatrix{N}) where {N}
-    n, p = size(G)
-
-    # TODO special case p = 1 or p = 2 ?
+function _single_quadrant_vertices_enum(G::AbstractMatrix{N}, sorted::Bool=true) where{N}
```

Comment:
I would set the default value of `sorted` to `false` and pass `true` explicitly at the (currently only) call site below.